timescaledb/getting-started/create-hypertable.md

```diff
@@ -81,10 +70,21 @@ effective use of memory and disk resources. By automatically and transparently
 partitioning on time, hypertables improve resource use. Queries and
 data-stores become more efficient.
 
+   COMMENT:
+   I feel like we probably don't need this section but wherever this section is, 
+   I think it would be cool to include a graphic that shows these benefits 
+   https://iobeam.slack.com/archives/C0J94TE4F/p1646682667481189?thread_ts=1646682487.374619&cid=C0J94TE4F
+-->
+
 ## Learn more about hypertables and chunks
 See the [Hypertable How To](/how-to-guides/hypertables) to learn more about
```

Comment:
To learn more about hypertables and best practices for configuring chunks, see ...
To learn why hypertables help with storing and query data, see ...